relay-server/src/statsd.rs

```diff
@@ -456,6 +458,7 @@ impl CounterMetric for RelayCounters {
             RelayCounters::Requests => "requests",
             RelayCounters::ResponsesStatusCodes => "responses.status_codes",
             RelayCounters::EvictingStaleProjectCaches => "project_cache.eviction",
+            RelayCounters::MetricBucketsParsingFailed => "metrics.buckets.parsing_failed",
```

Comment:
I put this here instead of [here](https://github.com/getsentry/relay/blob/master/relay-metrics/src/statsd.rs) because the statsd metric is emitted from envelopes.rs and not from `relay-metrics`. Not sure if it makes sense.